app/src/test/java/com/github/alfabravo2013/readyforexams/domain/home/ChecklistRepositoryTest.kt

```diff
@@ -0,0 +1,137 @@
+package com.github.alfabravo2013.readyforexams.domain.home
+
+import com.github.alfabravo2013.readyforexams.domain.models.Checklist
+import com.github.alfabravo2013.readyforexams.domain.models.toChecklistRepresentation
+import com.github.alfabravo2013.readyforexams.util.Result
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+
+internal class ChecklistRepositoryTest {
+    private val checklistLocalDataSource = mockk<ChecklistLocalDataSource>()
+    private val checklistRepository = ChecklistRepository(checklistLocalDataSource)
+
+    private val checklist = Checklist("name", listOf())
+    private val checklistRepresentation = checklist.toChecklistRepresentation()
+
+    @Nested
+    @DisplayName("When getChecklists")
+    inner class GetChecklistsTest {
+
+        @Test
+        @DisplayName("When no checklists have been added yet, Then return empty list")
+        fun getChecklistsEmptyList() {
+            every { checklistLocalDataSource.getChecklists() } answers { emptyList() }
+
+            val checklists = checklistRepository.getChecklists()
+
+            assertEquals(0, checklists.size)
+
+            verify(exactly = 1) { checklistLocalDataSource.getChecklists() }
+        }
+
+        @Test
+        @DisplayName("When 1 checklist has been added, Then return a list size == 1")
+        fun getChecklistsNonEmptyList() {
+            every { checklistLocalDataSource.getChecklists() } answers { listOf(checklist) }
+
+            val checklists = checklistRepository.getChecklists()
+
+            assertEquals(1, checklists.size)
+
```

Comment:
no empty line between the assertion and verification is needed. please review for all the places.